templates/help.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
```

Comment:
You currently have the same content shared across multiple templates. This goes against the DRY principle in software development called. Jinja has features that allow you to avoid this including "Template inheritance" (which will allow you to create a 'base' or 'layout' template) and "includes". You should use these here. 

You can read about this here: https://jinja.palletsprojects.com/en/2.11.x/templates/